go.mod

```diff
@@ -1,14 +1,20 @@
 module github.com/kyma-incubator/reconciler
 
-go 1.18
+go 1.21
+
+replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
+
+replace github.com/docker/docker => github.com/docker/docker v23.0.7-0.20230730020554-801e90549aac+incompatible
```

Comment:
To reviewer:
have to use this verison atm https://github.com/testcontainers/testcontainers-go/issues/1359#issuecomment-1657098312